lib/util/init.config.js

```diff
@@ -157,10 +157,10 @@ const Init = {
     /**
      * handles creation/update of all config file from the boilerplate
      *
-     * @param {string} versionBeforeUpgrade 'x.y.z'
+     * @param {string} properties config file's json
```

Comment:
this wouldnt work. we overwrite that object before we hit this method and hence would pass in the new version. please revert entire commit

Reply:
ok, but with my tests it worked exactly as expected, so lets run through one together. alternatively, i can revert and open a bug for fixing, but it makes it impossible for me to test the new init with auth class unless it is resolved

Reply:
will update to use version directly due to upgrade scenarios